templates/result-email-template.html

```diff
@@ -117,6 +117,14 @@ <h2>Suite duration</h2>
                 <td>{{ suite_run_time.total }}</td>
         </tr>
     </table>
+    <h2>Trigger CLI</h2>
```

Comment:
I am not sure if this is the right place. We are capturing the CLI options across different rows. I liked the way it was broken up and highlighted the important ones. If we are missing specific information like `hotfix`, it makes good sense to have it added. Putting the entire command after breaking it out looks out of place. These are my thoughts...